languages/ruby/reference/README.md

```diff
@@ -72,28 +80,15 @@ The Ruby concept exercises are based on concepts. The list below contains the co
   - unless
   - case
   - ternary operator
```

Comment:
The ternary operator is a form of `if` perhaps a sub-point.  But conditionals... flow control instead?  I don't know, thinking out loud.  Conditionals aren't helpful unless… right?  And Conditionals is not a thing like Module or Array or Hash, not an object.